.github/workflows/deps.yaml

```diff
@@ -12,25 +12,64 @@ on:
       - ".github/workflows/**"
       - "Makefile"
       - "extension_config_local.cmake"
+  workflow_call:
+    inputs:
+      duckdb-repo:
+        description: "The DuckDB repository to use"
+        required: false
+        type: string
+      duckdb-ref-env-var:
+        description: "Environment variable name that holds the ref in Makefile"
+        required: false
+        type: string
+      duckdb-ref-type:
+        description: "Reference type to check: release or tip"
+        required: false
+        type: string
 
 concurrency:
-  group: ${{ github.ref }}
+  group: "${{ github.workflow }}-${{ github.ref }}-${{ inputs.duckdb-repo }}"
   cancel-in-progress: true
 
 jobs:
+  get-latest-ref:
```

Comment:
Why are you doing this in deps i would expect it to be set in code by the time `deps.yaml` is run